system/lib/compiler-rt/lib/asan/asan_flags.cc

```diff
@@ -59,7 +65,11 @@ void InitializeFlags() {
     CommonFlags cf;
     cf.CopyFrom(*common_flags());
     cf.detect_leaks = cf.detect_leaks && CAN_SANITIZE_LEAKS;
+#if !SANITIZER_EMSCRIPTEN
+    // getenv on emscripten uses malloc, which we can't when using LSan.
+    // You can't run external symbolizers anyway.
```

Comment:
Can you elaborate on why external symbolizers can't be used? Is it just because there are no external processes in WebAssembly/Emscripten?

Reply:
Yes, we can't run external processes, so the whole code for that is not usable.